examples/platform/silabs/SilabsDeviceDataProvider.cpp

```diff
@@ -37,26 +34,39 @@ namespace Silabs {
 // using namespace chip::Credentials;
 using namespace chip::DeviceLayer::Internal;
 
-// TODO Remove once Commander supports (doesn't erase) NVM3 for 917
-#ifdef SIWX917_USE_COMISSIONABLE_DATA
-void SilabsDeviceDataProvider::setupPayload(uint8_t * outBuf)
+CHIP_ERROR SilabsDeviceDataProvider::generatePayload(uint8_t * outBuf)
```

Comment:
In what case do you still use this if you now have access to our factory provisioning mechanism? 

If you do still need it, it still should stay in a conditional #if. This code should not appear for products or for the EFR32 families